README.rst

```diff
@@ -103,7 +103,9 @@ requirements file, just run ``tox``.
 The project also contains JavaScript tests which can be run using
 [Karma](karma-runner.github.io). ``tox`` will run the JavaScript tests
 after the Python tests. You can run only the JavaScript tests using
-``docker-compose run web tox -e js``.
+``docker-compose run web tox -e js``, or do continuous javascript
```

Comment:
lol, I suppose I could spell it correctly, or at least the same it is four other times in the file.